ci/build_libtelio.py

```diff
@@ -302,6 +304,16 @@ def main() -> None:
         action="store_true",
         help="Generate python bindings with uniffi",
     )
+    build_parser.add_argument(
+        "--tcli",
+        action="store_true",
+        help="Include tcli package",
+    )
+    lipo_parser.add_argument(
+        "--tcli",
+        action="store_true",
+        help="Include tcli package",
+    )
```

Comment:
Why are these line duplicated?

Reply:
they add the tcli option for both build and lipo parsers. So that we can use the `--tcli` arg both for build and lipo commands.